servers/cromwell/Dockerfile.dev

```diff
@@ -0,0 +1,16 @@
+# Note: This is the dockerfile for development purposes
+
+FROM gcr.io/google-appengine/python
```

Comment:
Why does this use a different base image than the cromwell `Dockerfile`?

Reply:
I copied this from the master branch. so this was the current `Dockerfile` in master branch.
The reason that the new production `Dockerfile` using `openjdk` is that `rebuild_swagger.sh` requires `java`. So I have to use the `multi-stage` docker build to make the production image work.

Reply:
Got it! Thanks for the explanation!